app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  validates :first_name, presence: true, length: { in: 5..20 }
+  validates :last_name, presence: true, length: { in: 5..20 }
```

Comment:
Setting a minimum length of 5 characters is too much in my opinion. What about names like "Joe", "John", "Joe" or even "Al"? These users won't be able to sign up with the above-mentioned constraint. I'd strongly suggest to remove the length constraint to a minimum of two characters [`OPTIONAL`].